pages/api/Schema/reports/designationAccounts/designationAccounts.graphql

```diff
@@ -4,9 +4,7 @@ extend type Query {
 
 extend type Mutation {
   setActiveDesignationAccount(
-    accountListId: ID!
-    active: Boolean!
-    designationAccountId: ID!
+    input: SetActiveDesignationAccountInput!
   ): SetActiveDesignationAccountResponse!
```

Comment:
If you could make this return a `DesignationAccount`, the Apollo Client should be able to update its cache after the mutation and rerender the query.

That being said, it might be not be doable if you can't get all the fields you need from the REST PUT request. But hopefully you can.

Reply:
Any thoughts on making this return the same type as the query? You might be able to avoid having to do the manual cache updates you wrote.